### ISSUE TITLE
Gemini [Create images failed: we can't create your images right now]

### ISSUE THREAD

**Issue by djdoolky76** [Zero6992/chatGPT-discord-bot#501]
The chat prompt seems to be functioning properly, but when I try to generate an image, an error message appears.

![image](https://github.com/Zero6992/chatGPT-discord-bot/assets/30578879/92e26b28-1495-4e53-8089-ad6c6a19fc82)

**Comment by Zero6992** [comment]
What model are you using?

**Comment by djdoolky76** [comment]
I'm using Gemini Advanced

**Comment by lavanda131** [comment]
the same problem

**Comment by Zero6992** [comment]
Did you provided Google PSID into.env?

**Comment by djdoolky76** [comment]
> Did you provided Google PSID into.env?

Yes, and the chat prompt seems to be working fine, except for draw image

**Comment by Zero6992** [comment]
It might be some bug in gpt4free

Similar issue here: https://github.com/xtekky/gpt4free/issues/1671

**Comment by LaioX** [comment]
same problem here, tried gemini, bing and you.
Openai model show 
Something Went Wrong, try again later.
Error Message:No .har file found.
Any solution ?
